examples/pytorch/gat/gat.py

```diff
@@ -26,14 +28,15 @@ def __init__(self, attn_drop):
 
     def forward(self, nodes):
         a1 = torch.unsqueeze(nodes.data['a1'], 1)  # shape (B, 1, 1)
-        a2 = nodes.mailbox['a2'] # shape (B, deg, 1)
-        ft = nodes.mailbox['ft'] # shape (B, deg, D)
+        a2 = nodes.mailbox['a2']  # shape (B, deg, 1)
+        ft = nodes.mailbox['ft']  # shape (B, deg, D)
         # attention
         a = a1 + a2  # shape (B, deg, 1)
         e = F.softmax(F.leaky_relu(a), dim=1)
         if self.attn_drop != 0.0:
-            e = F.dropout(e, self.attn_drop)
-        return {'accum' : torch.sum(e * ft, dim=1)} # shape (B, D)
+            e = F.dropout(e, self.attn_drop, training=self.training)
```

Comment:
Instead of using nn.functional.dropout, let's change to a dropout layer, then we don't need to explicitly have `training=self.training`